client/src/main/java/com/alibaba/nacos/client/config/http/ServerHttpAgent.java

```diff
@@ -66,7 +67,7 @@ public HttpResult httpGet(String path, List<String> headers, List<String> paramV
                               long readTimeoutMs) throws IOException {
         final long endTime = System.currentTimeMillis() + readTimeoutMs;
 
-        boolean isSSL = false;
+        boolean isSSL = Boolean.getBoolean("com.alibaba.nacos.client.naming.tls.enable");
```

Comment:
I think the property name should not carry a 'naming', since the current module is Config.

Reply:
copy from com.alibaba.nacos.client.config.http.ServerHttpAgent.httpGet:70